### PR TITLE
referral-update

### PR DESCRIPTION
**Pull request by naomigreen**
Note: the second step has a condition that ends the form if you state "Was information and advice offered?" = no. For this reason I've moved the rest of the questions to the final step

To test
Form conditions:
Was information and advice offered? = yes 
Will this Contact lead to a Safeguarding Concern? = yes
Proceed to further case activities (provide details below) 
No Further Action

Designs
https://www.sketch.com/s/654c1c53-d967-4269-9507-25ca8423e730/a/pjwE0p
https://www.sketch.com/s/654c1c53-d967-4269-9507-25ca8423e730/a/VlbRyw
https://www.sketch.com/s/654c1c53-d967-4269-9507-25ca8423e730/a/gpybLr

Google form 
https://docs.google.com/forms/d/e/1FAIpQLSc4y4yjw6DpwWHN2fias1SEDo0lZZZNgN3M20Zy_p2He1rSuw/viewform

updatedReferralForm

### CODE REVIEW

File: pages/people/[id]/records/referral/[[...stepId]].tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+import type { User, Resident } from 'types';
+
+type FormData = Resident & User & Record<string, unknown>;
+
+const Referral = (): ReactElement => {
+  const { query } = useRouter();
+  const { user } = useAuth() as { user: User };
+  const onFormSubmit = useCallback(
+    // (person) => async (formData: FormData) =>
```
Comment (LBHABEBIC) [inline pages/people/[id]/records/referral/[[...stepId]].tsx:22]: do we still need these comments?

Comment (naomigreen) [reply pages/people/[id]/records/referral/[[...stepId]].tsx:22]: Confirm by Silvia that we're releasing it later. So we'll keep the function commented out for now.

**Comment by sirLisko** [comment]
The form looks good, I think it's worth double-checking what are the required fields, as at the moment we don't have any.

**Review by sirLisko — APPROVED** [review]
👍

**Pull request merged**